README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Community Edition and docker-compose
+  * Mac:
+    1. [Download](https://www.docker.com/community-edition#/download) and install Docker.
+    2. Start Docker by clicking on the application’s icon.
+  * Linux:
+    ```bash
+    curl -sSL https://get.docker.com/ | sudo bash
+    sudo usermod -aG docker $USER && newgrp docker
+    sudo systemctl start docker
+
+    sudo curl -L https://github.com/docker/compose/releases/download/1.11.1/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
```

Comment:
Sure, I'll change it. It'll probably get outdated again pretty soon, but might as well suggest the newest thing for now.